cypress/integration/not-authenticated.spec.ts

```diff
@@ -16,8 +14,9 @@ describe('Not authenticated access', () => {
   });
 
   it('should redirect when clicking the auth button', () => {
+    cy.wait(1000);
     cy.get('[data-cy=btn-authenticate]').click();
-
+    cy.wait(1000);
```

Comment:
Do you really need `waits`? Because on user-office-frontend we had a lot of waits but we removed almost all of them and we got much better results in testing time.

Reply:
Dunno, I had it sometimes fail without any explanation and after re-run everything worked so wonder what is happening inside cypress, after waiting a bit the number of false-positive fails stopped, so I'd like to keep it here.